orienteer-core/src/main/java/org/orienteer/core/tasks/OTaskSessionRuntime.java

```diff
@@ -33,6 +33,7 @@ public OTaskSessionRuntime() {
 	
 	public OTaskSessionRuntime(String sessionClass) {
 		persistedSession = new OTaskSessionPersisted(new ODocument(sessionClass));
+		persistedSession.save().getDocument().getDatabase().commit();
```

Comment:
Why commiting of document needed here? Idea was "do not save" document unless session started.